dim_checks/mozdata/fenix/active_users_aggregates/dim_checks.yaml

```diff
@@ -0,0 +1,41 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
+    slack: lvargas
+  slack_alerts:
+    enabled: true
+    notification_level: INFO
+    # TODO: maybe this could be made more granual
+    # and we could have different settings per channel?
+    notify_channel: true
+    notify:
+      channels:
+        # - kpi-monitoring-dim
+        - dim_test_run
+  tier: tier_1
+  partition_field: submission_date
+  dim_tests:
+    - type: custom_compare_metrics
+      title: DAU validation - Firefox Android (Fenix)
+      description: Validating that DAU sum in Active Users Aggregates is the same as unique client_id count in the live table.
+      params:
+        condition: dau_sum = distinct_client_count
+        # TODO: metric should be a list rather than a map
+        # to enable us an arbitary number of metrics in the comparison
+        metric_1:
+          name: dau_sum
+          query: |
+            SELECT SUM(dau),
+            FROM `{{ project_id }}.{{ dataset }}.{{ table }}`
+            WHERE
+              submission_date = "{{ params.partition }}"
+              AND channel = "release"
```

Comment:
Is there a reason to check only release or we plan to add the other channels later?